packages/autorest.typescript/src/restLevelClient/generateRestLevel.ts

```diff
@@ -121,6 +123,15 @@ export async function generateRestLevelClient() {
     generateFileByBuilder(project, buildPackageFile, rlcModels);
     // buildTsConfig
     generateFileByBuilder(project, buildTsConfig, rlcModels);
+  } else {
+    // update package.json if existing
+    const packageJsonContent = JSON.parse(await host.readFile("package.json"));
```

Comment:
@kazrael2119 it should work with readFile, could you have a verify?

Reply:
I have verified and this works well